src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -299,7 +299,6 @@ class WorkspaceInvitePage extends React.Component {
                                     multiline
                                     containerStyles={[styles.workspaceInviteWelcome]}
                                     value={this.state.welcomeNote}
-                                    placeholder={this.getWelcomeNotePlaceholder()}
```

Comment:
I think we should rename the function to getWelcomeNote.

Reply:
@rushatgabhane, done.